src/universal/components/SignInPage/SignInEmailPasswordForm.js

```diff
@@ -28,9 +28,7 @@ const Form = styled('form')({
   flexDirection: 'column'
 });
 
-const FieldsContainer = styled('div')({
-  marginBottom: '2rem'
-});
+const Block = styled('div')(({margin, width}) => ({margin, width}));
```

Comment:
I like this utility you added! If you think it'll be a useful pattern going forward, let's add it to `universal/components/`

Reply:
Good point…I may just do that.